pkg/issuer/acme/acme.go

```diff
@@ -92,6 +92,22 @@ func New(issuer v1alpha1.GenericIssuer,
 	}, nil
 }
 
+// uaRoundTripper implements the http.RoundTripper interface and adds a User-Agent
+// header. Note that this is a stopgap until upstream `crypto/acme` adds a
+// facility for setting User-Agent.
+
+type uaRoundTripper struct {
+	http.RoundTripper
+	ua string
+}
+
+const Version = "v0.2.3"
```

Comment:
We can use `pkg/until.AppVersion` here instead to save it being maintained in multiple places